lib/montage/query.rb

```diff
@@ -8,9 +8,33 @@ class Query
     include Montage::Support
 
     attr_accessor :query
+    attr_reader :schema
 
-    def initialize
-      @query = { filter: {} }
+    # Initializes the query instance
+    #
+    # * *Attributes* :
+    #   - +schema+ -> The name of the schema you wish to query.  Alphanumeric
+    #     characters and underscores are allowed.
+    #   - +query+ -> A query hash containing desired options
+    # * *Returns* :
+    #   - A valid Montage::Query instance
+    # * *Raises* :
+    #   - +InvalidAttributeFormat+ -> If the declared schema is not a string or
+    #     contains non-alphanumeric characters.  Underscore ("_") is allowed!
+    #
+    def initialize(params = {})
+      @schema = params[:schema]
+      @query = { "$schema" => @schema, "$query" => { "$filter" => {} } }
```

Comment:
If you look at the [ReQON home page](https://github.com/dmpayton/reqon#reql-query-object-notation), the query object is an array, as is the filter object. It should look like:
```ruby
@query = { 
  "$schema" => @schema, 
  "$query" => [
    ["$filter", []]
  ] 
}
```

Because of all the nested arrays, I think it makes sense also to break this out into multiple lines, just so it's more readable.